onnxruntime/core/session/abi_session_options.cc

```diff
@@ -340,3 +340,9 @@ ORT_API_STATUS_IMPL(OrtApis::SetDeterministicCompute, _Inout_ OrtSessionOptions*
   return nullptr;
   API_IMPL_END
 }
+
+ORT_API_STATUS_IMPL(OrtApis::SessionOptionsSetLoadCancellationFlag, _Inout_ OrtSessionOptions* options,
+                    _In_ bool is_cancel) {
+  options->value.SetLoadCancellationFlag(is_cancel);
+  return nullptr;
```

Comment:
may be unneeded today, but perhaps it may throw someday.

```suggestion
  API_IMPL_BEGIN
  options->value.SetLoadCancellationFlag(is_cancel);
  return nullptr;
  API_IMPL_END
```